sfepy/discrete/fem/meshio.py

```diff
@@ -2822,7 +2902,7 @@ def read(self, mesh, omit_facets=True, **kwargs):
                 pass
 
             else:
-                output('skipping unknown entity: %s' % line)
+                # output('skipping unknown entity: %s' % line)
```

Comment:
This line can be simply removed. The other readers do not print such information. (Alternative: add `verbose` argument to `read()`.)

Reply:
I removed it.